presto-main/src/main/java/io/prestosql/testing/MaterializedResult.java

```diff
@@ -250,22 +250,22 @@ private static void writeValue(Type type, BlockBuilder blockBuilder, Object valu
             blockBuilder.appendNull();
```

Comment:
Are we going to tackle more like that?
```
kokosing@m16:~/presto$ git grep '(Number) ' | wc -l
141
```

Reply:
I think we should. Do you want to file an issue or make a PR?